modules/ROOT/pages/manage-apis-with-service-mesh.adoc

```diff
@@ -21,15 +21,23 @@ To apply policies:
 . Navigate to *API Manager* and select your API instance.
 . From the left navigation, *select Policies > Apply New Policy*:
 +
-image::select-policies-for-service-mesh.png[]
+.The diagram shows how to apply a policy to your API.
+image::select-policies-for-service-mesh.png[Applying Policies to APIs]
++
+<1> The page where you specify a policy for your API is displayed.
+<2> Select any one of the policies to apply.
 +
 . Select any of the following policies to secure your APIs: 
 * *Client ID Enforcement* 
 * *JWT Validation* 
 * *Rate Limit* 
 * *Rate Limit SLA-Based* 
-
-image::apply-client-enforcement-policy-service-mesh.png[]
++
```

Comment:
This diagram is at a lower level in the list than the previous one and I'm not seeing a reason for that.

Reply:
Not sure how I can change it, so keeping it as is for now.